payments-core/src/main/java/com/stripe/android/googlepaylauncher/StripeGooglePayContract.kt

```diff
@@ -46,32 +43,3 @@ internal class StripeGooglePayContract :
         }
     }
 }
-
-internal fun GooglePayLauncherResult.Error.getErrorResourceID(): Int? {
-    return when (googlePayStatus) {
-        Status.RESULT_SUCCESS -> null
-        Status.RESULT_INTERNAL_ERROR, Status.RESULT_CANCELED, Status.RESULT_DEAD_CLIENT, null ->
-            R.string.stripe_google_pay_error_internal
-        else -> {
-            when (googlePayStatus.statusCode) {
-                CommonStatusCodes.API_NOT_CONNECTED, // "The client attempted to call a method from an API that failed to connect."
-                CommonStatusCodes.CANCELED, // -> "The result was canceled either due to client disconnect or PendingResult.cancel()."
-                CommonStatusCodes.DEVELOPER_ERROR, // -> "The application is misconfigured."
-                CommonStatusCodes.ERROR, // -> "The operation failed with no more detailed information."
-                CommonStatusCodes.INTERRUPTED, // -> "A blocking call was interrupted while waiting and did not run to completion."
-                CommonStatusCodes.INVALID_ACCOUNT, // -> "The client attempted to connect to the service with an invalid account name specified."
-                CommonStatusCodes.SERVICE_DISABLED, // -> "This constant is deprecated. This case handled during connection, not during API requests. No results should be returned with this status code."
-                CommonStatusCodes.SERVICE_VERSION_UPDATE_REQUIRED, // -> " This constant is deprecated.This case handled during connection, not during API requests . No results should be returned with this status code."
-                CommonStatusCodes.SUCCESS, // -> "The operation was successful.SUCCESS_CACHE The operation was successful, but was used the device's cache."
-                CommonStatusCodes.RESOLUTION_REQUIRED, // -> "Completing the operation requires some form of resolution."
-                CommonStatusCodes.INTERNAL_ERROR -> R.string.stripe_google_pay_error_internal
-                CommonStatusCodes.NETWORK_ERROR -> R.string.stripe_failure_connection_error
-                CommonStatusCodes.SIGN_IN_REQUIRED -> R.string.stripe_failure_reason_authentication
-                CommonStatusCodes.TIMEOUT -> R.string.stripe_failure_reason_timed_out
```

Comment:
These are not really the same as what the message says. SIGN_IN_REQUIRED is returned if the user is not logged in to GPay and doesn't log in when prompted. TIMEOUT is on the connection to Google Play Services. I think it's less confusing to treat them as generic errors. We're still logging the GPay error code and error message from the Throwable.